content/pt-br/docs/reference/glossary/garbage-collection.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Coleta de lixo
```

Comment:
@MrErlison here is the problem, please, can you change this term "Coleta de lixo" to "Coletor de lixo".